inventory/inventory

```diff
@@ -17,7 +17,8 @@ mesos_slaves
 load_balancers
 
 [all:vars]
-ansible_python_interpreter="PATH=/home/core/bin:$PATH python"
+# The only user able to ssh to a newly created Rackspace instance is root (at least initially), so we need to update the python PATH to accommodate for that.
+ansible_python_interpreter="PATH=/home/core/bin:/root/bin:$PATH python"
```

Comment:
Yes it is unfortunately, regardless of the image type / OS used (eg. ubuntu,centos,coreos etc.), in Rackspace the first time you try to ssh in one of the machines you can only do so as root. You can change that behaviour from within the box after you ssh as root initially of course. See here for more info: https://support.rackspace.com/how-to/basic-cloud-server-security/